integration

```diff
@@ -256,6 +256,7 @@
   "delphiki/hass-pronote",
   "denpamusic/homeassistant-plum-ecomax",
   "dext0r/yandex_smart_home",
+  "denysdovhan/ha-yasno-outages",
```

Comment:
There is no country defined https://github.com/denysdovhan/ha-yasno-outages/blob/main/hacs.json

Reply:
Fixed in [`v0.3.7`](https://github.com/denysdovhan/ha-yasno-outages/releases/tag/v0.3.7)